src/main/java/mate/academy/rickandmorty/Application.java

```
@@ -1,12 +1,23 @@
 package mate.academy.rickandmorty;
 
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.service.CharacterExternalApiService;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
-public class Application {
+@RequiredArgsConstructor
+public class Application implements CommandLineRunner {
+    private final CharacterExternalApiService externalApiService;
 
     public static void main(String[] args) {
         SpringApplication.run(Application.class, args);
+
+    }
+
+    @Override
+    public void run(String... args) throws Exception {
+        externalApiService.fetchCharacters();
```

Suggestion (optional): If you prefer the application to start even when the external API is temporarily unavailable, consider adding error handling/logging here to catch exceptions from fetchCharacters() and decide whether to continue startup or fail fast. Current behavior (propagating exceptions) may be acceptable if you want startup to fail on missing data.